transform/files_test.go

```diff
@@ -0,0 +1,94 @@
+package transform
+
+import "testing"
+
+func TestIsFileOf(t *testing.T) {
+	types := []kind{company, partner, facility}
+	cases := []struct {
```

Comment:
My suggestion is to change the variable  from `cases` to `casesTest` to don't conflict with reserved word `case`.

Reply:
Good point!